specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2023-06-01/managedClusters.json

```diff
@@ -4557,6 +4557,34 @@
           },
           "title": "The upgrade channel for auto upgrade. The default is 'none'.",
           "description": "For more information see [setting the AKS cluster auto-upgrade channel](https://docs.microsoft.com/azure/aks/upgrade-cluster#set-auto-upgrade-channel)."
+        },
+        "nodeOSUpgradeChannel": {
```

Comment:
Maybe update the description of the enum "node-image" under "upgradeChannel" as well? Currently it's different from the preview API version
> "Automatically upgrade the node image to the latest version available. Consider using nodeOSUpgradeChannel instead as that allows you to configure node OS patching separate from Kubernetes version patching"

Reply:
Yes, please update the description.

Reply:
Thanks updated!